Seeing

```
Error: ❌  DCO check failed.

Commits missing the 'Signed-off-by:' footer:
f070dba

🛠 **Quick fix (make ONE signed commit):**
    git fetch origin
    git reset --soft origin/main
    git commit -s -m "<your message>"
    git push --force-with-lease

🔧 **Fix individual commits:**
    git rebase -i origin/main --exec "git commit --amend -s --no-edit"
    git push --force-with-lease

💡  Or edit the commit message in GitHub UI and add:
    Signed-off-by: Your Name <email@example.com>
Error: Process completed with exit code 1.
```

see https://github.com/openai/codex?tab=readme-ov-file#developer-certificate-of-origin-dco.

Could you resolve?

--- a/codex-cli/src/cli.tsx
+++ b/codex-cli/src/cli.tsx
@@ -1,5 +1,9 @@
 #!/usr/bin/env node
 
+// Hack to suppress deprecation warnings (punycode)
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+(process as any).noDeprecation = true;
+
 import type { AppRollout } from "./app";
 import type { ApprovalPolicy } from "./approvals";
 import type { CommandConfirmation } from "./utils/agent/agent-loop";

